### PR TITLE
Updating EntityUrl output object in urlResolvers query

### PR DESCRIPTION
**Pull request by VinothKumar361**
## Purpose of this pull request

This pull request (PR) updates the EntityUrl output object in urlResolvers query.As seen in this page (https://github.com/magento/magento2/blob/5f3b86ab4bd3e3b94e65429fed33f748d29c1bbe/app/code/Magento/UrlRewriteGraphQl/etc/schema.graphqls#L12) there is no url attribute in EntityUrl output object also the redirectCode attribute is missing in the docs.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/graphql/queries/url-resolver.html

## Links to Magento source code

-  https://github.com/magento/magento2/blob/5f3b86ab4bd3e3b94e65429fed33f748d29c1bbe/app/code/Magento/UrlRewriteGraphQl/etc/schema.graphqls#L12
https://github.com/magento/magento2/blob/5f3b86ab4bd3e3b94e65429fed33f748d29c1bbe/app/code/Magento/UrlRewriteGraphQl/Model/Resolver/EntityUrl.php#L115

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/graphql/queries/url-resolver.md [hunk 1]
```diff
@@ -63,7 +66,7 @@ Attribute |  Data Type | Description
 `id` | Int | The ID assigned to the object associated with the specified `url`. This could be a product ID, category ID, or page ID.
 `relative_url` | String | The internal relative URL. If the specified  `url` is a redirect, the query returns the redirected URL, not the original.
 `type` | UrlRewriteEntityTypeEnum | The value of `UrlRewriteEntityTypeEnum` is one of PRODUCT, CATEGORY, or CMS_PAGE.
-`url` | String | The URL to resolve. Magento stores product and category URLs with the `.html` extension.  CMS URLs do not contain the extension.
+`redirectCode` | Int | 301 or 302 HTTP code for url permanent or temporary redirect or 0 for the 200 no redirect.
```
Comment (rogyar) [inline src/guides/v2.3/graphql/queries/url-resolver.md]: Let's make this statement simpler.

```suggestion
`redirectCode` | Int | 301 or 302 HTTP status code for URL permanent or temporary redirect. Contains 0 when there's no redirect.
```

File: src/guides/v2.3/graphql/queries/url-resolver.md [hunk 2]
```diff
@@ -26,7 +26,9 @@ The following query returns information about the URL containing `joust-duffle-b
   urlResolver(url: "joust-duffle-bag.html") {
     id
     relative_url
+    redirectCode
     type
+
```
Comment (keharper) [inline src/guides/v2.3/graphql/queries/url-resolver.md]: Remove the blank line

**Comment by VinothKumar361** [comment]
@keharper @rogyar changes made as per your requests.

**Comment by keharper** [comment]
@rogyar I rewrote the description for `redirectCode`. Please review the PR again

**Comment by keharper** [comment]
running tests

**Pull request merged**